### PR TITLE
docs: rbac authz doc

### PR DESCRIPTION
**Pull request by JigarJoshi**

### CODE REVIEW

File: docs/concepts/authnz/index.md [hunk 1]
```diff
@@ -53,7 +53,83 @@ issuer, audience, and expiration.
 
 ## Authorization
 
-Tigris supports AWS’s IAM policies mechanism as the authorization system.
+Tigris supports Role-Based-Access-Control (RBAC) & AWS’s IAM policies mechanism
+as the authorization system. Tigris dashboard allows users to create access key
+with a role. This role is mapped to a bucket. Following is the permission model.
+
+| Operation                          | NamespaceAdmin | Editor | ReadOnly |
```
Comment (himank) [inline docs/concepts/authnz/index.md:60]: Should we just call it `Admin`?

File: docs/concepts/authnz/index.md [hunk 2]
```diff
@@ -53,7 +53,83 @@ issuer, audience, and expiration.
...
+Note:
+
+- In addition to S3, Tigris offers specific custom APIs. Tigris will seamlessly
+  incorporate newly added methods into appropriate roles without causing
+  disruptions.
+
+- In addition to above, bucket owner is allowed to perform all the operations on
+  their own bucket.
+
+In addition to the role, NamespaceAdmin users can also further customize the
```
Comment (himank) [inline docs/concepts/authnz/index.md:131]: May be just `Admin`

**Review by himank — APPROVED** [review]

File: docs/concepts/authnz/index.md (hunk 1 above)
Comment (JigarJoshi) [reply docs/concepts/authnz/index.md]: renamed

File: docs/concepts/authnz/index.md (hunk 2 above)
Comment (JigarJoshi) [reply docs/concepts/authnz/index.md]: renamed

**Review by ovaistariq — APPROVED** [review]

**Pull request merged**